scipy/stats/stats.py

```diff
@@ -205,6 +205,9 @@
 
 
 def _chk_asarray(a, axis):
+    if len(np.array(a).shape) == 0:
```

Comment:
This would typically be checked as `if np.array(a).ndim == 0:`, and similarly in the other two occurrences.